modules/ROOT/pages/use-tags-in-visualizer.adoc

```diff
@@ -28,10 +28,10 @@ By default, when you first view a node using Anypoint Visualizer, it does not ha
 
 . Select the application node to tag. +
 You can hold the shift key to select multiple nodes to apply tags to all the selected nodes at once. +
-_Result:_ The Customization panel for the node appears in the panel on the right as shown in the following image:
+_Result:_ The *Customization* section for the node appears in *Details/Monitoring* card as shown in the following image:
```

Comment:
same comment as before:
"appears in the Details tab of the node's card"

Reply:
Accepted.